src/main/java/wooteco/subway/member/application/MemberService.java

```diff
@@ -24,11 +27,25 @@ public MemberResponse findMember(Long id) {
         return MemberResponse.of(member);
     }
 
+    public MemberResponse findMemberByPayLoad(String payLoad) {
+        return MemberResponse.of(memberDao.findByEmail(payLoad));
+    }
+
     public void updateMember(Long id, MemberRequest memberRequest) {
         memberDao.update(new Member(id, memberRequest.getEmail(), memberRequest.getPassword(), memberRequest.getAge()));
     }
 
     public void deleteMember(Long id) {
         memberDao.deleteById(id);
     }
+
+    public MemberResponse findMemberByTokenRequest(TokenRequest tokenRequest) {
+        try{
+            Member member = memberDao.findByTokenRequest(tokenRequest);
+            return MemberResponse.of(member);
+        } catch (EmptyResultDataAccessException exception){
+            throw new UserLoginFailException();
+        }
```

Comment:
```suggestion
        try {
            Member member = memberDao.findByTokenRequest(tokenRequest);
            return MemberResponse.of(member);
        } catch (EmptyResultDataAccessException exception) {
            throw new UserLoginFailException();
        }
```

컨벤션을 위반하셨습니다. IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
커밋 되는 순간 자동으로 설정되도록 해놨을 텐데 자꾸 안되어있네요. 주의하겠습니다.